hack/testing/logging.sh

```diff
@@ -142,7 +142,7 @@ os::cmd::expect_success 'oadm router'
 ######### logging specific code starts here ####################
 
 os::cmd::expect_success "oadm new-project logging --node-selector=''"
-os::cmd::expect_success "oc project logging"
+os::cmd::expect_success "oc project logging > /dev/null"
```

Comment:
just out of curiosity is this just to suppress the output that we may already be on this project? e.g. this has no impact on the expect_success